### PR TITLE
Add expectedCommandLineArguments parameter to create/update request API

### PR DESCRIPTION
**Pull request by Calvinp**
Adds a parameter to request API for expected command line arguments. These might get filled in as your default parameters in the run now dialog.
When running a task for the first time, if set, this will take precedence over Singularity's guess that you want to run the task with the same arguments as the previous run. When rerunning a task, that task's arguments will take precedence.

### CODE REVIEW

**Comment by Calvinp** [comment]
Expected Command Line Arguments moved to deploy config.

**Comment by ssalinas** [comment]
:+1: looks good, can you fix the merge conflict with master when you get a chance

File: SingularityUI/app/models/Request.coffee [hunk 1]
```diff
@@ -457,6 +457,18 @@ class Request extends Model
             app.caughtError()
             callback() # Don't actually error if we can't find previous args, just don't populate
 
+    getCommandLineArgs: (callback) =>
+        console.log "Get command line args"
```
Comment (Calvinp) [inline SingularityUI/app/models/Request.coffee]: :-1: I managed to backdoor my commit hook.

**Comment by Calvinp** [comment]
Merge conflict fixed.

**Comment by Calvinp** [comment]
Renamed to expectedRunTaskNowCommandLineArguments

**Comment by ssalinas** [comment]
We are going to set this one aside and come back to this issue with a different strategy, closing for now (already took the changes out of hs_staging)

**Pull request closed without merging**